gui/default/index.html

```diff
@@ -285,7 +285,7 @@ <h4 class="panel-title">
                   <span ng-switch-when="idle"><span class="hidden-xs" translate>Up to Date</span><span class="visible-xs">&#9724;</span></span>
                   <span ng-switch-when="syncing">
                     <span class="hidden-xs" translate>Syncing</span>
-                    ({{syncPercentage(folder.id)}}%)
+                    ({{syncPercentage(folder.id)}}%, {{syncRemaining(folder.id)}})
```

Comment:
I was trying to make it concise! Could add "to go" I suppose, but I think it should be pretty clear when you watch it for a few seconds and it goes down.